### DISCUSSION TITLE
Changing the default avo-assets path?

### DISCUSSION THREAD

**Discussion by emiranda-beon** [avo-hq/avo#3252] [Help] [answered]
Is there a way for us to specify a custom assets path to serve `avo.base.css`, `avo.base.js`, `avo-dashboard-assets` [...] from?

**Comment by adrianthedev** [comment]
hmm, there might be a way.
Can you please describe your use-case a bit more? What is the problem you are facing?

**Comment by therabidbanana** [comment]
We could use this feature to be able to support multiple avo instances within the same domain. At our company we have several Rails backend services sharing the same domain via load balancer rules, and a separate load balancer for "admins" that exposes those same Rails services but has a Cloudflare Google authentication system in front. This allows relatively seamless linking back and forth in the admin panels since theres a lot of shared data between them.

We've been using routing rules like "/admin/labs" and "/admin/learning" to separate out the routes by service type, but they end up sharing the same assets and needing to be deployed in sync when there's an Avo update because only one of these services serves the offical `avo-assets` route. (This isn't an attempt to avoid paying for licenses or anything, we're just kinda stuck on this legacy service layout while we continue to deprecate the old admin panels)

**Comment by adrianthedev** [comment, marked as answer]
Thanks for raising awareness about this. We figured that most people aren't affected by how we ship assets, but hearing you it helped.

This is going to be fixed in Avo 4 by https://github.com/avo-hq/avo/pull/3971

We are using more Rails-specific patterns and won't have the `avo-assets` path anymore.
All the assets will pass through the regular asset pipeline using sprockets.
